controls/nist_ocp4.yml

```diff
@@ -1463,16 +1463,20 @@ controls:
   - high
   - moderate
 - id: AC-12(1)
-  status: pending
+  status: partial
   notes: |-
-    Section a: A control response is planned. Progress can be tracked via:
+    OpenShift provides a logout capability for user-initiated communications sessions whenever
+    authentication is used to gain access to the system when the default kubeadmin account is disabled.
+    
+    The OpenShift does not have the logout capability when cert is being used with the default kubeadmin account,
+    therefore we need to check if kubeadmin account is disabled.
```

Comment:
What do you think about using the text about certificate logins and revocation we have here: https://github.com/JAORMX/content/blob/be396cf36803f9890d6947d9688a74b3a9f535d7/applications/openshift/authentication/idp_is_configured/rule.yml#L42 ?

Reply:
Thanks for the suggestion, I will update the content there